qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -126,13 +126,19 @@ class WebEngineSearch(browsertab.AbstractSearch):
     def __init__(self, parent=None):
         super().__init__(parent)
         self._flags = QWebEnginePage.FindFlags(0)
+        self.num_of_searches = 0
 
     def _find(self, text, flags, callback, caller):
         """Call findText on the widget."""
         self.search_displayed = True
+        self.num_of_searches += 1
 
         def wrapped_callback(found):
             """Wrap the callback to do debug logging."""
+            self.num_of_searches -= 1
```

Comment:
Do we need to worry about a race condition on this decrement (@The-Compiler)? I'm not sure how the python callbacks work, so this might not be needed.